api/api-doc/storage_service.json

```diff
@@ -728,6 +728,14 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"query"
+                  },
+                  {
+                     "name":"offstrategy",
+                     "description":"When set to true, trigger offstrategy compaction (and return immediately)",
```

Comment:
If we don't provide a better explanation of what this "offstrategy compaction" means, we need to create documentation about it - Google didn't find any. As a user I would have no idea what exactly will happen when I "trigger offstrategy compaction" or why or when I would want to do it.

Reply:
I opened https://github.com/scylladb/scylla/issues/9162 as a follow-up.